src/app/pages/common/landing/landing.component.ts

```diff
@@ -33,7 +33,9 @@ export class LandingComponent implements OnInit {
         this.appServ.loggedInUserInfo["persona"],
         this.appServ.loggedInUserInfo["id"]
       ];
-      if (persona == "seller") {
+      if (persona == "admin") {
+        this.router.navigate(["admin", "seller-list"]);
```

Comment:
Can this `if-else` ladder use the `persona` as the first argument? (something like meta programming)?

Reply:
Also, the 2nd element in the array can be a lookup from a map? or maybe the whole array itself can be a lookup from a map?

Reply:
Fixed now.